### ISSUE TITLE
NTR: Abnormal salivary cortisol

### ISSUE THREAD

**Issue by callahantiff** [obophenotype/human-phenotype-ontology#5628]
**Preferred term label:**
- Abnormal salivary cortisol level
- Decreased Salivary cortisol level 
- Increased Salivary cortisol level 

<br>

**Synonyms**  
- **Abnormal salivary cortisol level**: Abnormal salivary cortisol  
- **Decreased Salivary cortisol level**: High salivary cortisol level; Hypercortisolism; Cushing syndrome; Increased cortisol production  
- **Increased Salivary cortisol level**: Low salivary cortisol level; Low to undetectable salivary cortisol; Decreased cortisol production; Glucocorticoid insufficiency; Addison's disease    

<br>

**Definition (free text, please give PubMed ID)**
- **Abnormal salivary cortisol level**: An abnormality of the concentration of cortisol in saliva    
- **Decreased Salivary cortisol level**: Abnormally reduced concentration of cortisol in saliva
- **Increased Salivary cortisol level**: Abnormally increased concentration of cortisol in saliva  

[PMID:19602555](https://www.ncbi.nlm.nih.gov/pubmed/19602555) 
[PMID:18280810](https://www.ncbi.nlm.nih.gov/pubmed/18280810)

<br>

**Parent term (use hpo.jax.org/app)**  
Literature suggests that saliva provides a more sensitive measure of cortisol than serum or plasma. Looking at the current hierarchy, perhaps _Abnormal salivary cortisol level_ could be added as a subclass of _Abnormality of the endocrine system (HP_0000818)_.

<br>

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**  

Result | OMIM | ORPHANET
:--: | :--: | :--:  
Low salivary cortisol | [OMIM:103230](https://omim.org/entry/103230)<br>[OMIM:240200](https://omim.org/entry/240200) | [ORPHA:85138](https://www.orpha.net/consor/cgi-bin/Disease_Search.php?lng=EN&data_id=11626&Disease_Disease_Search_diseaseGroup=Addison-s-disease&Disease_Disease_Search_diseaseType=Pat&Disease(s)/group%20of%20diseases=Addison-disease&title=Addison%20disease&search=Disease_Search_Simple)  
High salivary cortisol | [OMIM:219090](https://omim.org/entry/219090) | [ORPHA:96253](https://www.orpha.net/consor/cgi-bin/Disease_Search.php?lng=EN&data_id=12827&Disease_Disease_Search_diseaseGroup=Cushing-s-disease&Disease_Disease_Search_diseaseType=Pat&Disease(s)/group%20of%20diseases=Cushing-disease&title=Cushing%20disease&search=Disease_Search_Simple)

<br>

**Your nano-attribution (ORCID)**  
https://orcid.org/0000-0002-8169-9049

**Comment by pnrobinson** [comment]
Adding terms to a new created parent, Abnormal salivary metabolite concentration
